src/speed.c

```diff
@@ -25,12 +20,6 @@ const float angular_acceleration_defaults[NUM_MODES] = {32. * PI, 32. * PI,
 const float max_end_linear_speed_defaults[NUM_MODES] = {.5, .55, 0.61, 0.72};
 const float max_linear_speed_defaults[NUM_MODES] = {.5, .55, 0.61, 0.72};
 const float max_linear_speed_run_defaults[NUM_MODES] = {1., 1.5, 2., 2.5};
-const float max_angular_speed_defaults[NUM_MODES] = {2.35 * PI, 3. * PI,
-						     4. * PI, 6. * PI};
-const float turn_linear_speed_defaults[NUM_MODES] = {0.4, 0.448, 0.566, 0.670};
-const float turn_radius_defaults[NUM_MODES] = {0.07, 0.07, 0.07, 0.07};
-const float turn_t0_defaults[NUM_MODES] = {219., 166., 124., 88.};
-const float turn_t1_defaults[NUM_MODES] = {291., 260., 208., 176.};
 
 uint8_t speed_configuration;
```

Comment:
Could you include a comment with the units and meaning of the structure fields?

Reply:
e5f10cf3b1dca58a4f0f82d182a06ab78fb5b51c